pkgs/phps.nix

```diff
@@ -16,6 +16,21 @@ let
       pcre2 = if (prev.lib.versionAtLeast args.version "7.3") then prev.pcre2 else prev.pcre;
 
       phpAttrsOverrides = attrs: {
+        patches =
+          let
+            upstreamPatches =
+              attrs.patches or [];
+
+            ourPatches =
+              prev.lib.optionals (prev.lib.versions.majorMinor args.version == "7.2") [
+                # Patch from: https://bugs.php.net/bug.php?id=76826
```

Comment:
Would be nice to include a short explanation in the comment.

Reply:
Done.